README.md

```diff
@@ -3,18 +3,19 @@
 [![Travis CI Build Status](https://travis-ci.org/dbohdan/remarshal.svg?branch=master)](https://travis-ci.org/dbohdan/remarshal)
 [![AppVeyor CI Build Status](https://ci.appveyor.com/api/projects/status/github/dbohdan/remarshal?branch=master&svg=true)](https://ci.appveyor.com/project/dbohdan/remarshal)
 
-Convert between JSON, MessagePack, TOML, and YAML.  When installed,
+Convert between JSON, MessagePack, TOML, YAML, and CBOR.  When installed,
```

Comment:
This is optional, but since the formats are ordered alphabetically, I would prefer you to put CBOR first in the lists in documentation and code.

Reply:
I didn’t want to presume that my contribution would “supersede” anything else. :)

Reply:
Fixed.